source/extensions/filters/http/grpc_json_transcoder/json_transcoder_filter.cc

```diff
@@ -58,6 +61,9 @@ const Http::LowerCaseString& trailerHeader() {
   CONSTRUCT_ON_FIRST_USE(Http::LowerCaseString, "trailer");
 }
 
+constexpr uint32_t HttpBodyFieldNumber = 2;
```

Comment:
inferer those from protobuf lib?

Reply:
Can you advice if there is a presence of `ProtobufLengthDelimitedField`-like constant in protobuf, please? It looks like it's hardcoded as `2` everywhere in the libprotobuf codebase, and all potential helper functions are in `internal` namespace.

Reply:
Yeah I see that is only in internal::WireFormat, can you move this constexpr to the function you use it (there's only one), and add a comment where it is from?

Reply:
Done, added additional comment to already present link to wire format documentation.